server/index.js

```diff
@@ -20,5 +21,81 @@ app.get('/api/alsoBought/:id', (req, res) => {
   });
 });
 
+// Get all stock records
+app.get('/api/alsoBought/', (req, res) => {
+  Stock.readAll((err, stocks) => {
+    if (err) {
+      res.sendStatus(400);
+    } else {
+      res.status(200).json(stocks);
+    }
+  });
+});
+
+//Show the stock record in the db for the specific id
+app.get('/api/alsoBought/:id/show', (req, res) => {
+  Stock.readOne(req.params.id, (err, stock) => {
+    if (stock) {
+      res.status(200).json(stock);
+    } else {
+      res.sendStatus(404);
+    }
+  });
+});
+
+//Create a new stock record
+app.post('/api/alsoBought/', (req, res) => {
+  Stock.create(req.body.stock, (err, newStock) => {
+    if (err) {
+      res.sendStatus(400);
+    } else {
+      res.status(201).json(newStock);
+    }
+  });
+});    
+
+//Update a existing stock record
+app.put('/api/alsoBought/:id', (req, res) => {
+  Stock.update(req.params.id, req.body.stock, (err, stock) => {
```

Comment:
I believe that update and delete both look for objects in the first parameter, so this might not be the right syntax (also goes for line 69)